packages-ts/gauntlet-terra-contracts/src/commands/abstract/executionWrapper.ts

```diff
@@ -1,7 +1,21 @@
 import AbstractCommand, { makeAbstractCommand } from '.'
 import { Result } from '@chainlink/gauntlet-core'
 import { TerraCommand, TransactionResponse } from '@chainlink/gauntlet-terra'
-import { AccAddress, MsgExecuteContract } from '@terra-money/terra.js'
+import { AccAddress, LCDClient } from '@terra-money/terra.js'
+import { logger, prompt } from '@chainlink/gauntlet-core/dist/utils'
+
+export type ExecutionContext<Input, ContractInput> = {
```

Comment:
Where are these `<Input, ContractInput>` types defined?

Reply:
In the commands. See set billing for example: https://github.com/smartcontractkit/chainlink-terra/blob/develop/packages-ts/gauntlet-terra-contracts/src/commands/contracts/ocr2/setBilling.ts#L7

Reply:
Aha, these are actually generic type definitions but it's confusing (to me at least) as they are named the same as other concrete types. It wouldn't be as confusing if defined as `export type ExecutionContext<A, B> = {` for example.

No specific suggestions now. Types are improving step by step, and we'll continue to improve them as we go.